### PR TITLE
Scale to multiple containers

### PR DESCRIPTION
**Pull request by alexle-dev**

### CODE REVIEW

**Comment by Wowu** [comment]
Thank you @alexle-dev, I'll look into it!

By the way, can you share what is your case of running multiple copies of same container on a single machine?

**Comment by alexle-dev** [comment]
@Wowu I have a React app behind a Traefik reverse proxy, and I scale it to multiple instances in order to balance the load between all available cores. Traefik takes care of the loadbalancing for me, and it automatically detects new containers.

**Comment by jokester** [comment]
To add two cents, I also run scaled containers with docker-compose, mostly for service availability.

With docker-compose one can set a ranged port mapping: `ports: [ '18080-18090:8080']` (this may be difficult to support as a shell script though) or [let multiple container share 1 host port](https://iximiuz.com/en/posts/multiple-containers-same-port-reverse-proxy/). I have been using the latter for a few times.

File: docker-rollout [hunk 1]
```diff
@@ -72,84 +74,119 @@ scale() {
...
-  if [[ $(echo "$OLD_CONTAINER_ID" | wc -l) -gt 1 ]]; then
-    echo "Service '$SERVICE' has more than one container running. Make sure 'scale' is set to 1." >&2
-    echo "Fix with: $COMPOSE_COMMAND up --detach --scale $SERVICE=1 $SERVICE" >&2
-    exit 1
+  # shellcheck disable=SC2086
+  if [[ "$($COMPOSE_COMMAND $COMPOSE_FILES ps -q "$SERVICE")" == "" ]]; then
+    echo "==> Service '$SERVICE' is not running. Starting the service."
+    $COMPOSE_COMMAND $COMPOSE_FILES up --detach --scale="$SERVICE=$SCALE" --no-recreate "$SERVICE"
+    exit 0
   fi
```
Comment (Wowu) [inline docker-rollout:89]: Was it intended to check `! ($COMPOSE_COMMAND $COMPOSE_FILES ps -q "$SERVICE")` and then `"$($COMPOSE_COMMAND $COMPOSE_FILES ps -q "$SERVICE")" == ""` separately? Isn't a single `if` sufficient in this case?

**Review by Wowu — COMMENTED** [review]
Thank you again for your contribution! I have a question about the implementation.

**Comment by Wowu** [comment]
@alexle-dev I decided to remove `--scale` flag, as we can detect scale from the running number of container. Let me know if setting scale manually is required for your use case 😃

**Comment by Wowu** [comment]
@martiGIT released in [v0.3](https://github.com/Wowu/docker-rollout/releases/tag/v0.3)

**Pull request merged**